docs/stylesheets/extra.css

```diff
@@ -103,3 +103,15 @@
   color: var(--md-code-fg-color);
   background-color: var(--md-code-bg-color);
 }
+
+.md-nav__title {
+  display: none;
+}
+
+.md-nav__link {
+  margin-top: 0.3em;
+}
+
+.md-nav--secondary .md-nav__link {
+  margin-top: 0.5em;
+}
```

Comment:
I'll admit that I'm hesitant to add stuff like this for a few reasons: (1) we need to test it against all responsive breakpoints, not just on desktop; (2) it's not robust to updates to MkDocs or Material for MkDocs (what if they change the styling, or even the class names -- we'd have no way of knowing that this broke other than visual inspection).

Reply:
I understand the concern, but the default navigation is going to be a problem. I'm not sure what to suggest as an alternative.

Reply:
Ok. There is a very good chance that this will silently break at some point. But I think you hear me.

At the very least, could I request that you: (1) add comments to each change in the CSS, and (2) add screenshots for all breakpoints?

Reply:
I definitely hear you. I'm happy to take an alternative approach if there is one. Otherwise, I think this is (1) not a huge deal if there's a regression, though I could see how changes could interact badly if / when we have more custom CSS (2) a temporary solution until we revisit documentation rendering as a whole.

Reply:
(Sent a DM about the breakpoints)